config/src/config/consensus_config.rs

```diff
@@ -209,48 +230,49 @@ impl Default for ConsensusConfig {
+                    max_sending_block_txns_override: MAX_SENDING_BLOCK_TXNS,
+                    max_sending_block_bytes_override: 5 * 1024 * 1024,
                     backpressure_proposal_delay_ms: 400,
                     max_txns_from_block_to_execute: None,
                 },
                 PipelineBackpressureValues {
                     back_pressure_pipeline_latency_limit_ms: 2000,
-                    max_sending_block_txns_override: 1000,
-                    max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
+                    max_sending_block_txns_override: MAX_SENDING_BLOCK_TXNS,
+                    max_sending_block_bytes_override: 5 * 1024 * 1024,
                     backpressure_proposal_delay_ms: 500,
-                    max_txns_from_block_to_execute: Some(400),
+                    max_txns_from_block_to_execute: None,
                 },
+                // with other pipeline backpressure, only later start reducing block size
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 2300,
+                    back_pressure_pipeline_latency_limit_ms: 3000,
                     max_sending_block_txns_override: 1000,
                     max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
                     backpressure_proposal_delay_ms: 500,
-                    max_txns_from_block_to_execute: Some(150),
+                    max_txns_from_block_to_execute: None,
                 },
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 2700,
+                    back_pressure_pipeline_latency_limit_ms: 4000,
                     max_sending_block_txns_override: 1000,
                     max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
                     backpressure_proposal_delay_ms: 500,
-                    max_txns_from_block_to_execute: Some(50),
+                    max_txns_from_block_to_execute: Some(200),
                 },
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 3100,
+                    back_pressure_pipeline_latency_limit_ms: 5000,
                     max_sending_block_txns_override: 1000,
                     max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
                     backpressure_proposal_delay_ms: 500,
-                    max_txns_from_block_to_execute: Some(20),
+                    max_txns_from_block_to_execute: Some(30),
                 },
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 3500,
+                    back_pressure_pipeline_latency_limit_ms: 6000,
                     max_sending_block_txns_override: 1000,
                     max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
                     backpressure_proposal_delay_ms: 500,
```

Comment:
> Also, what is the logic behind the chosen numbers?

nothing more than gradual reduction as values change. we could've also just had formula instead of multiple discrete values, not a big deal either way

> In this block of PipelineBackpressureValues I don't understand why the gas is not limited. Wouldn't that affect the execution time better than # of txs?

number of txns is known upfront (i.e. when block is being created), and how block gas limit affects the block is only know after execution. So if proposer wants to create smaller block for smaller messages, # of txns need to be adjusted.

Once we have execution pool, and latency overhead of transaction being included in the block, but hitting the limit, being drastically reduced, plan is to change most backpressures to reduce block gas limit, instead of transactions. (i.e. max_sending_block_txns_override will be in txns, and max_txns_from_block_to_execute will be changed to be max_block_gas_limit_override)